contrib/completion/zsh/_docker-compose

```diff
@@ -330,6 +330,8 @@ _docker-compose() {
         '(- :)'{-v,--version}'[Print version and exit]' \
         '(-f --file)'{-f,--file}'[Specify an alternate docker-compose file (default: docker-compose.yml)]:file:_files -g "*.yml"' \
         '(-p --project-name)'{-p,--project-name}'[Specify an alternate project name (default: directory name)]:project name:' \
+        '--x-networking[(EXPERIMENTAL) Use new Docker networking functionality. Requires Docker 1.9 or later.]' \
```

Comment:
Ya, we had to make it opt-in for compose because we don't want to force every compose user to upgrade to 1.9 immediately.